app/main.py

```diff
@@ -1,8 +1,53 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
It is a common practice to place each argument in the `__init__` on a new line, so that opening and closing parenthesis would not have any arguments on the same line. Follow this approach in the future.